bridge/src/client/cli/client_command.rs

```diff
@@ -103,16 +103,47 @@ impl ClientCommand {
         })
     }
 
+    pub fn get_funding_amounts_command() -> Command {
+        Command::new("get-funding-amounts")
+            .short_flag('m')
+            .about("Get minimum required amounts for the funding UTXOs (to be used in testing)")
+            .after_help("This is useful when creating the funding UTXOs for the peg-in and peg-out graphs.")
+    }
+
+    pub async fn handle_get_funding_amounts(&self) -> io::Result<()> {
+        const INITIAL_AMOUNT: u64 = 2 << 20; // 2097152
+
+        println!("Minimum required input amounts");
+        println!("------------------------------");
+        println!(
+            "Peg-in graph - 'peg-in depost' tx input:    {} SAT (spendable by [DEPOSITOR])",
```

Comment:
I thought that it's subtraction because of the minus (-). Can we modify the formar of those messages?
```suggestion
            "[PEG-IN graph] 'peg-in depost' tx input:    {} SAT (spendable by [DEPOSITOR])",
```

Reply:
I removed the graph info instead.